pkgs/by-name/h2/h2o/package.nix

```
@@ -23,13 +23,13 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "h2o";
-  version = "2.3.0-rolling-2025-08-22";
+  version = "2.3.0-rolling-2025-08-29";
```

```suggestion
  version = "2.3.0-unstable-2025-08-29";
```
That is the standard naming for it. Also we do not want to update the package every day. r-ryantm I think usually waits at least 1 week or so.

---

Hmm okay, didn't know that.

---

Only thing that I find noteworthy here is that you did 6 commits but this could be one. Is there a deeper reason behind that?